app/src/main/res/layout/shopping_list_item.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/item_read_only"
+        android:layout_width="290dp"
```

Comment:
Please don't hardcode dimensions in dp. That's why you have wrap_content and match_parent, or constraints in ConstraintLayout.